engine/src/main/java/org/terasology/input/cameraTarget/PlayerTargetSystem.java

```diff
@@ -66,6 +76,11 @@ public void update(float delta) {
             if (charComp != null) {
                 Vector3f dir = player.getViewDirection();
                 float maxDist = charComp.interactionRange;
+                FirstPersonHeldItemMountPointComponent heldItemMountPoint = player.getCameraEntity().getComponent(FirstPersonHeldItemMountPointComponent.class);
```

Comment:
In my reading of the code, a 'PlayerTargetSystem` class does not seem to be associated with the eye ball. `PlayerTargetSystem` disambiguating between using gaze-based targeting and hand-based targeting is something I would expect.

I feel like the next logical step here is to create `GazeBasedPlayerTargetSystem` and `TrackedControllerPlayerTargetSystem` or something. As above, we can push this through for now and I can add that to my bigger "vr subsystem" PR, or we can nuke this and just wait for that one.

Anyway, everyone's reading of a piece of code can be different, so I'm up for revising it in a way that makes more sense. I just can't figure out a better place to put this. Suggestions welcome. :)